### DISCUSSION TITLE
Inquiry on Multi-Monitor Handling and Display Visibility in Waybar with Hyprdots

### DISCUSSION THREAD

**Discussion by vcopaf8** [prasanthrangan/hyprdots#1447] [General]
Hello,

I am relatively new to using Arch Linux and have recently decided to delve into the world of dotfiles, choosing Hyprdots based on its excellent features and community recommendations. First off, I’d like to commend you for the impressive work on this project; it’s been quite inspiring for a newcomer like me.

I have a couple of specific concerns that I hope you can help me address:

1. **Multi-Monitor Handling**: I’ve noticed that upon system reboot, the connection with multiple displays automatically works, which is fantastic. However, I encounter difficulties when trying to configure them manually. Previously, I successfully used `arandr` and `xrandr`, but they seem to have stopped functioning correctly after installing Hyprdots. Could you guide me on whether I need any additional settings to manage the screens manually?

2. **Visibility in Waybar**: Currently, I can’t seem to find a way to identify which screen I am on using Waybar. Is there a setting or command that could enable this functionality?

Any assistance or advice would be greatly appreciated, and again, thank you for the excellent work on the project.

Best regards,

**Comment by kRHYME7** [comment]
re1:  X11 apps borke, use [wlr-randr instead](https://github.com/emersion/wlr-randr) 
       personal recommendation using a  GUI: https://github.com/nwg-piotr/nwg-displays
But ` hyprctl monitors ` should be enough     
![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/bdca309f-aab9-4095-a8d3-8233f823cbd5)

I assumed you did not read the beautiful Hyprland wiki, https://wiki.hyprland.org/Configuring/Monitors/

re2: Not yet for now, waybar will be displayed on all monitors. there is a pending PR though. 

Goodluck!

**Reply by vcopaf8** [reply]
Thank you very much for your prompt and helpful response!

Regarding the multi-monitor handling, I appreciate the suggestion to use `wlr-randr` and the GUI tool from the nwg-displays project. I will definitely give them a try. Also, I’ll make sure to read through the Hyprland wiki more thoroughly to better understand the configurations—thanks for pointing that out!

As for Waybar, I understand that it's still in progress and I look forward to the updates. I’ll keep an eye on the pending PR.

Thanks again for your guidance and support. I’m excited to continue exploring and configuring Hyprdots!
